src/cmd-buildextend-installer

```diff
@@ -148,15 +148,32 @@ def generate_iso():
                     tarinfo.mode = 0o0644
                 return tarinfo
 
-            # Install binaries from the grub2-efi and shim rpms (installed
-            # in our COSA container). Manually construct the tarball to ensure
-            # proper permissions and ownership
+            tmpimageefidir = tempfile.TemporaryDirectory()
+            ostreeefidir="/usr/lib/ostree-boot/efi/EFI"
+            for folder in ['BOOT', 'redhat', 'fedora']:
```

Comment:
This is totally fine for now, but in theory we should support having EFI dirs with other vendor names too (there's at least one other that comes to mind...).  To do that we'd have to do something more like parse the output of `ls -R` and find one not empty.

Reply:
Should be fixed by a7f0347 - fetch a list of subfolders in `EFI` and attempt to checkout them all